.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '10.x'
+
+    - run: npm i
+
+    - run: npm start build
+
+    - run: npx semantic-release@15
```

Comment:
Won't you need to pass an npm token in your env variables? Not sure

Reply:
Neither was I. Tough to check on this one until after it's merged 😂

Reply:
I've added a `NPM_TOKEN` secret to both this repo and prettier-eslint-cli